numerics/elliptic_functions.cpp

```diff
@@ -1,16 +1,36 @@
 ﻿
 #include "numerics/elliptic_functions.hpp"
 
+#include <tuple>
+
 #include "glog/logging.h"
+#include "numerics/combinatorics.hpp"
 #include "numerics/elliptic_integrals.hpp"
+#include "numerics/polynomial.hpp"
+#include "numerics/polynomial_evaluators.hpp"
+#include "quantities/elementary_functions.hpp"
 #include "quantities/numbers.hpp"
 
 namespace principia {
+
+using quantities::Sqrt;
+
 namespace numerics {
 namespace {
 
 void JacobiSNCNDNReduced(double u, double mc, double& s, double& c, double& d);
 
+// Maclaurin series for Fukushima b₀.  These are polynomials in m that are used
+// to build a polynomial in u₀².  The index gives the degree of the polynomial.
```

Comment:
> that are used ~to build~ *as coefficients* of a polynomial in u₀²

> The index gives the degree of the polynomial

The index should give the corresponding power of u₀² .

Reply:
Done.